src/utils/colonyActions.ts

```diff
@@ -550,17 +551,31 @@ export const formatRolesTitle = (roles: ActionUserRoles[]) => {
   };
 };
 
+const getChangelogItem = (
+  {
+    isMotion: actionIsMotion,
+    transactionHash,
+    pendingColonyMetadata,
+  }: ColonyAction,
+  colonyMetadata?: ColonyMetadata | null,
+) => {
+  const metadataObject = actionIsMotion
```

Comment:
colonyMetadata won't yet reflect the change if it's a motion, so we look at the pending metadata instead